### PR TITLE
Added Review on Hindi Translations

### PR DESCRIPTION
**Pull request by sshiv5768**
> Make sure you go through the [Contributing guide](https://docs.legesher.io/the-official-things/contributing-guidelines).

> Please delete options that are not relevant.

### CODE REVIEW

**Comment by sshiv5768** [comment]
@madiedgar , please check it.

**Comment by madiedgar** [comment]
Ah @sshiv5768 this is so great! 👏 Thank you so much for your hard work on translating all of these. A couple things to do before I can merge this in: 
- make the base branch that you're merging your commits into to be `hindi-translation` instead of `master`. (when Hindi gets all of the official reviews completed, it'll then be merged into master with the official translation 😄. For now, we're making changes on the branch.)
- I _just_ merged a new translation review to the `hindi-translation` branch. You might need to pull the latest version. Don't lose any of your work though! 
- For the keywords that already have a translation there, just add yours (with any commentary) in the comments. Please don't delete any of the translations because when its time for the final review, we'll need to look at all of the words. 👍 I believe the only programming language that this was done to is Python.

Let me know if you have any questions. 👍

File: locale/hi.yml [hunk 1]
```diff
@@ -0,0 +1,1813 @@
+# Files in the locale directory are used for localization.
+# If you want to use locales other than English, follow the instructions in legesher-translations guide.
+# https://legesher.readme.io/docs/translation-guide
+#
+# THANK YOU FOR MAKING LEGESHER LARGER THAN US.
+
+# NOTE: "language-abbreviation"
+language-abbreviation: "hi"
```
Comment (madiedgar) [inline locale/hi.yml:8]: Good catch 👍

**Review by madiedgar — CHANGES_REQUESTED** [review]
I looked at your other PR and saw some great things. I just can't remember exactly what they were since this PR is being merged to master and its treating it like a new file. Left some changes in the comments. 👍

File: locale/hi.yml (hunk 1 above)
Comment (sshiv5768) [reply locale/hi.yml:8]: Is it for me?😅

Comment (sshiv5768) [reply locale/hi.yml:8]: So, @madiedgar what should I do now? Should I change this PR?

**Pull request closed without merging**